### ISSUE TITLE
Suport of MacStadium cloud

### ISSUE THREAD

**Issue by dfad1ripe** [dgzlopes/cloud-detect#21]
Hi, it would be great to see the MacStadium cloud supported.
The one supposed test is to `curl https://ipinfo.io/` and check if .org field of output is 'AS395336 MacStadium, Inc.'. Not sure if good API calls exist for that, as entire their API seems to require authentication.

**Comment by kshivakumar** [comment]
@dfad1ripe Just curious about your use case. Do you deploy your application on different cloud providers?

**Comment by dfad1ripe** [comment]
Yes, and I need to know in the application which cloud it runs in. For that, I use the JSON generated by that page at ipinfo.io, and analyze the .org field reported there, this satisfies my needs by now. To be honest, I do not plan to use your module in the nearest future, just suggesting that you might add one more provider to the list. :)
